### PR TITLE
LPD-41509 Revert "LPD-37385 Use template literal and use native Intl.…

### PR DESCRIPTION
**Pull request by bakayattila**
…DateTimeFormat instead of moment"

This reverts commit c036b580ea4964725ec3f34bd419b46a7907e903.

### CODE REVIEW

**Comment by bakayattila** [comment]
ci:test:content-management-playwright

**Comment by bakayattila** [comment]
ci:test:content-management-playwright

File: modules/test/playwright/tests/document-library-web/fileEntry.spec.ts [hunk 1]
```diff
@@ -124,20 +124,34 @@ test(
...
+			{
+				day: 'numeric',
+				hour: 'numeric',
+				hour12: true,
+				minute: 'numeric',
+				month: 'numeric',
+				year: '2-digit',
+			}
+		).format(new Date(scheduleDate))}.`;
+
```
Comment (boton) [inline modules/test/playwright/tests/document-library-web/fileEntry.spec.ts]: We can do it only with one variable message 😄 


```ts
		const message = `Success:${title} will be published on ${new Intl.DateTimeFormat(
			'en-US',
			{
				day: 'numeric',
				hour: 'numeric',
				hour12: true,
				minute: 'numeric',
				month: 'numeric',
				year: '2-digit',
			}
		).format(new Date(scheduleDate))}.`;

//... expect condition
message || message.replace(',', '')

```

**Review by boton — CHANGES_REQUESTED** [review]
I buy the idea but I suggest some changes inline.

File: modules/test/playwright/tests/document-library-web/fileEntry.spec.ts (hunk 1 above)
Comment (attilabakay) [reply modules/test/playwright/tests/document-library-web/fileEntry.spec.ts]: will do after the test run

**Comment by boton** [comment]
ci:test:content-management-playwright

**Pull request closed without merging**